app/src/main/java/com/pwhs/quickmem/presentation/app/classes/add_folder/AddFolderToClassViewModel.kt

```
@@ -0,0 +1,61 @@
+            AddFolderToClassUIAction.RefreshFolders -> {
+                TODO()
+            }
+
+            AddFolderToClassUIAction.SaveClicked -> {
+                addFolderToClass()
+            }
+        }
+    }
+
+    private fun addFolderToClass() {
+    }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)